src/webapp/public/locales/de/translation.json

```diff
@@ -23,7 +23,14 @@
           "timer_fade_volume": "Fade volume",
           "toggle_output": "Audio-Ausgang umschalten",
           "sync_rfidcards_all": "Alle Audiodateien und Karteneinträge synchronisieren",
-          "sync_rfidcards_change_on_rfid_scan": "Aktivierung ändern für 'on RFID scan' "
+          "sync_rfidcards_change_on_rfid_scan": "Aktivierung ändern für 'on RFID scan'",
+          "next": "Nächster Song",
+          "pause": "Pause",
+          "play": "Abspielen",
+          "previous": "Vorheriger Song",
+          "shuffle": "Shuffle aktivieren/deaktivieren",
+          "repeat": "Wiederholen aktivieren/deaktivieren",
+          "toggle": "Abspielen/Pause"
```

Comment:
"Abspielen/Pause umschalten" ? Like the other translations for toogles?

Reply:
No strong preference. Will change it.